FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,59 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 	
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
+		client.get(from: url) {[weak self] response in
+			guard self != nil else { return }
+			switch response {
+			case let .success((data, response)):
+				do {
+					let items = try FeedImageMapper.map(data: data, response: response)
+					completion(.success(items))
+				} catch {
+					completion(.failure(Error.invalidData))
```

Comment:
You can compelete with the caught `error`since the `FeedImageMapper.map` throws an `RemoteFeedLoader.Error.invalidData`:

```suggestion
					completion(.failure(error))
```